### ISSUE TITLE
Don't Convert Frames to Jpeg in Collab

### ISSUE THREAD

**Issue by Limeysoda1** [jantic/DeOldify#214]
The frames in the video colourization are full of JPEG artifacts which are especially noticable if you are using it to colourize older and lower resolution material.

**Comment by Limeysoda1** [comment]
They also create alot of haloing which is very undesirable.

**Comment by jantic** [comment]
I'll illustrate what we're dealing with here, with something that's pretty middle of the road in terms of source material quality. 

Here's an example of a render using the current high quality jpeg frames:

![00001_old](https://user-images.githubusercontent.com/179759/79805031-1f8dda80-831a-11ea-963b-71efa02f63bc.jpg)

And here's that same frame using png:

![00001](https://user-images.githubusercontent.com/179759/79805049-27e61580-831a-11ea-87f6-91c8bab5eb1b.png)

I do see artifacts that are in the jpeg that aren't present in png, and that's chiefly noticeable around the watermark. That's to be expected. But also the difference in the size of these files is 10x! This becomes a concern precisely when using Colabs because this adds up quickly on VMs that have very limited resources.

Basically my position on what I'm providing with the open source DeOldify is this:  You have something to experiment with and have fun with, but it's not aiming to be production quality stuff.  It's free, and it's  aiming to be low maintenance.  

Especially when the trade off in resources required is steep and the quality gain is fairly minimal- I'll opt for the low resource option.  If you want to fork it and create something that does lossless conversions- great, go ahead!  But I don't think it's worth the tradeoff here.

One more thing to note:  Haloing actually comes from the model, if you're talking about the thick outlines that form around objects sometimes.
